ethereum/eth/src/test/java/tech/pegasys/pantheon/ethereum/eth/sync/fastsync/FastSyncChainDownloaderTest.java

```diff
@@ -163,7 +182,12 @@ public void recoversFromSyncTargetDisconnect() {
 
     ethProtocolManager.handleDisconnect(bestPeer.getPeerConnection(), TOO_MANY_PEERS, true);
 
-    secondBestPeer.respondWhile(shorterResponder, () -> !result.isDone());
+    secondBestPeer.respondWhile(
+        shorterResponder,
+        () -> {
+          LockSupport.parkNanos(1000000);
+          return !result.isDone();
+        });
```

Comment:
I have a serious aversion to hard coded waits in tests so would be good to bring in https://github.com/ajsutton/pantheon/commit/0c7a625d4f7863d415a13d373358a0004e8590b9 if possible.

Reply:
done